collector/collection_status.go

```diff
@@ -3,8 +3,8 @@ package collector
 import (
 	"github.com/golang/glog"
 	"github.com/prometheus/client_golang/prometheus"
-	mgo "gopkg.in/mgo.v2"
-	"gopkg.in/mgo.v2/bson"
+	mgo "github.com/globalsign/mgo"
```

Comment:
does the mgo version not have the GetParameters call?

Reply:
if we don't change the dependency, it causes build failures since our gtm fork uses globalsign

Reply:
the other PR with the change was merged, so if you rebase we're all set with that :)